community/i3-gaps/build

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh -e
+
+set -u
+
+export CHOST="${CHOST:-x86_64-pc-linux-musl}"
```

Comment:
KISS doesn't use `CHOST` and only supports `x86_64-pc-linux-musl`. Change the `cd` to go directly to `x86_64-pc-linux-musl`.